### DISCUSSION TITLE
How to reverse default behavior of legend onItemClick?

### DISCUSSION THREAD

**Discussion by faizalami** [apexcharts/apexcharts.js#2823] [Q&A]
Hello, the current behavior of legend onItemClick is to hide the selected series and keep the other to display, but what I need is the opposite, I need to only display the series that I clicked and hide the other.

I found a bad solution that use looping and call [hideSeries method](https://apexcharts.com/docs/methods/#hideSeries) for each series names that must to be hidden, but of course this solution will be bad at performance, the chart will be re render each hideSeries called.

So, is there any method to hide mass/batch series? or how to do that?

why something as simple as series: {data: [...], name: 'a series', **show: false**} didn't exist?

**Comment by alon-loris** [comment]
I also need the reverse behavior, which is much more intuitive. Is there any solution for this?
